### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- Some minor refactors

### CODE REVIEW

File: pyrgg/params.py [hunk 1]
```diff
@@ -133,6 +133,10 @@
 
 PYRGG_CONFIG_SAVE_ERROR_MESSAGE = "[Error] Failed to save config file!"
 
+PYRGG_CONFIG_LIST_MESSAGE = "Config files detected in current directory are listed below :"
```
Comment (sadrasabouri) [inline pyrgg/params.py:136]: `Config files detected in the current directory are listed below:`

File: pyrgg/params.py [hunk 2]
```diff
@@ -133,6 +133,10 @@
...
+
+PYRGG_CONFIG_LOAD_MESSAGE = "Press the config index to load or any other keys to start with new one :"
```
Comment (sadrasabouri) [inline pyrgg/params.py:138]: `Press the config index to load or any other keys to start with the new one:`

File: test/functions_test.py [hunk 3]
```diff
@@ -201,7 +201,7 @@
 >>> load_config("test123456789")
 [Error] Failed to load config file!
 >>> loaded_config = check_for_config(input_func_conf_test1)
-Config files detected in current directory are listed below:
+Config files detected in current directory are listed below :
```
Comment (sadrasabouri) [inline test/functions_test.py:204]: Also, fix them accordingly.

File: test/functions_test.py [hunk 4]
```diff
@@ -210,7 +210,7 @@
 >>> loaded_config["output_format"] == input_data_["output_format"]
 True
 >>> loaded_config = check_for_config(input_func_conf_test2)
-Config files detected in current directory are listed below:
+Config files detected in current directory are listed below :
```
Comment (sadrasabouri) [inline test/functions_test.py:213]: Also, fix them accordingly.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Minor docstring challenges.

File: test/functions_test.py (hunk 4 above)
Comment (sepandhaghighi) [reply test/functions_test.py]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389

File: test/functions_test.py (hunk 3 above)
Comment (sepandhaghighi) [reply test/functions_test.py]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389

File: pyrgg/params.py (hunk 1 above)
Comment (sepandhaghighi) [reply pyrgg/params.py]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389

File: pyrgg/params.py (hunk 2 above)
Comment (sepandhaghighi) [reply pyrgg/params.py]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/a48ecb418570b7cfc9e3a251396cd319a43ffe33

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**